sys/net/gnrc/netif/Kconfig

```diff
@@ -45,8 +45,8 @@ config GNRC_NETIF_NONSTANDARD_6LO_MTU
     depends on USEMODULE_GNRC_NETIF_6LO
     help
         Enables the usage of non standard MTU for 6LoWPAN network interfaces.
-        This is non compliant with RFC 4944 and might not be supported by other
-        implementations.
+        This is non compliant with RFC 4944, RFC 7668 and might not be
```

Comment:
Same here
```suggestion
        This is non compliant with RFC 4944 and RFC 7668 and might not be
```

Reply:
Wanted to avoid an "and ... and" sentence. I changed it!

Reply:
In that case

```
        This is non compliant with RFC 4944 and RFC 7668. It might not be
```

might have also worked. But let's not continue messing around :-)

Reply:
True. 
That's what happens when you still try to touch as less than possible :D Even with such small things.